src/engine/strat_engine/setup.rs

```diff
@@ -198,7 +198,21 @@ pub fn get_blockdevs(pool_uuid: PoolUuid,
                     RangeAllocator::new(actual_size,
                                         segment_table.get(bda.dev_uuid()).unwrap_or(&vec![]))?;
 
-                blockdevs.push(StratBlockDev::new(*device, devnode.to_owned(), bda, allocator));
+                let bd_save = pool_save
```

Comment:
This is the only place where we have ready access to the blockdev's former devnode, if different from current. Is that worth recording? It's pretty evanescent, because once the pool metadata has been written, access to the prior devnode will be lost for good.

Reply:
As mentioned in another comment, I think it's only interesting from an error-reporting perspective. (Or maybe, if we find two blockdevs with the same dev_uuid, we tie-break using former devnode? bleh.) I'm not sure former devnode is very useful, but I'm not sure it isn't, either.